lib/content_location_filter.js

```diff
@@ -18,19 +18,26 @@ module.exports = (hyper, req, next, options) => {
             request: req
         }).uri;
         const attachLocation = (res) => {
-            if (res.status !== 301 && res.status !== 302) {
-                res.headers = res.headers || {};
-                Object.assign(res.headers, {
-                    'content-location': baseUri
-                        + new URI(req.uri.path.slice(2))
-                        + mwUtil.getQueryString(req)
-                });
+            res.headers = res.headers || {};
+            if (res.status === 301 || res.status === 302
```

Comment:
I might be missing something.. what would the pros / cons be of handling CORS redirects here?